.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Maybe add yarn.lock here too?

Reply:
if we ever have dependencies, we want to commit the yarn.lock. The reasons I removed it is because initially there were some dependencies but I ended up not needing them

Reply:
I don't think you should commit the yarn.lock in libraries. If tests can fail with any combination of valid semvers of your dependencies, then the packages that depend on you may observe those.

Reply:
The reason to commit the yarn.lock is if you want to ensure that your package is only ever built using a specific version of it's deps for reliability, but if your package is a library you don't actually have that guarantee. That's my understanding anyway